03week/towersOfHanoi.js

```diff
@@ -7,36 +7,54 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+//This is the initial setting at the beginning of towersOfHanoi.  Part of premade code
 let stacks = {
   a: [4, 3, 2, 1],
   b: [],
   c: []
 };
 
-function printStacks() {
+//This function prints the stacks on to the screen.  Part of premade code
+let printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
+//printBeginningStacks, reset the var stacks to its original position, nest stacks inside
+const printBeginningStacks=(stacks)=> {}
+  let stacks = {
+    a: [4, 3, 2, 1],
+    b: [],
+    c: []
+  };
+//movePiece(), Moves the last index of the startStack to the last index of the endStack, lastindexof method
+const movePiece=(startstack, endStack)=> {
+    endStack.push(startstack.lastindexof)
```

Comment:
lastIndexOf is being called incorrectly. It's a method, so you need to call it as a function and give it at least one argument. Please consult the docs https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/lastIndexOf